trufflescripts/add_token_pair.js

```diff
@@ -1,7 +1,9 @@
 /* eslint no-console:0, no-multi-spaces:0, prefer-destructuring:1 */
+const { updateExchangeParams } = require('./utils/contracts')(artifacts)
```

Comment:
If you are requiring `utils/contracts` may as well import `deployed`

Reply:
ok